src/test/java/website/frontrow/board/EnemyTest.java

```diff
@@ -59,4 +62,138 @@ public void testTick()
         //An enemy might move
         assertEquals(0.25, u.getLocation().getX(), 2);
     }
+    
+    /**
+     * Tests the setter for wallCollision.
+     */
+    @Test
+    public void testSetWallColision()
+    {
+    	Enemy enemy = new Enemy(super.p, null);
+    	assertFalse(enemy.getWallCollision());
+    	
+    	enemy.setWallCollision(true);
+    	assertTrue(enemy.getWallCollision());
+    }
+    
+    /**
+     * Tests the function onWallCollision.
+     */
+    @Test
+    public void testOnWallColision()
+    {
+    	Enemy enemy = new Enemy(super.p, null);
+    	assertFalse(enemy.getWallCollision());
+    	
+    	enemy.onWallCollision();
+    	assertTrue(enemy.getWallCollision());
+    }
+    
+    /**
+     * Tests the getter for tickCounter.
+     */
+    @Test
+    public void testGetTickCounter()
+    {
+    	Enemy enemy = new Enemy(super.p, null);
+    	assertEquals(enemy.getTickCounter(), 0);
+    }
+    
+    /**
+     * Tests the setter for tickCounter.
+     */
+    @Test
+    public void testSetTickCounter()
+    {
+    	Enemy enemy = new Enemy(super.p, null);
+    	assertEquals(enemy.getTickCounter(), 0);
+    	
+    	enemy.setTickCounter(10);
+    	assertEquals(enemy.getTickCounter(), 10);
+    }
+    
+    /**
+     * Tests the getter for random.
+     */
+	@Test
+    public void testGetRandom()
+    {
+    	Enemy enemy = new Enemy(super.p, null);
+    	// Not assertEquals, because that apparently doesn't 
+    	// accept two floats.
+    	assertTrue(enemy.getRandom() == 1.00f);
+    }
+	
+	/**
```

Comment:
No, I'll remove the tabs and replace them with spaces, like the rest of the class.